src/array_api_stubs/_2022_12/fft.py

```diff
@@ -18,13 +35,13 @@ def fft(
     Parameters
     ----------
     x: array
-        input array. Should have a floating-point data type.
+        input array. Should have a complex-valued floating-point data type.
     n: int
         length of the transformed axis of the output. If
 
-        - ``n`` is greater than the length of the input array, the input array is zero-padded to length ``n``.
-        - ``n`` is less than the length of the input array, the input array is trimmed to length ``n``.
-        - ``n`` is not provided, the length of the transformed axis of the output must equal the length of the input along the axis specified by ``axis``.
+        - ``n`` is greater than the length of the input array along ``axis``, the input array along ``axis`` must be zero-padded to length ``n``.
+        - ``n`` is less than the length of the input array along ``axis``, the input array along ``axis`` must be trimmed to length ``n``.
```

Comment:
@leofang Sorry for my ignorance, but "trimmed" in this case means the last `N-n` elements are removed?

Reply:
Let's address this in a small follow-on PR.